src/Symfony/Component/Serializer/Annotation/Groups.php

```diff
@@ -35,7 +35,7 @@ class Groups
      */
     public function __construct(array $data)
     {
-        if (!isset($data['value']) || !$data['value']) {
+        if (empty($data['value'])) {
```

Comment:
Even on this one actually: https://github.com/symfony/symfony/pull/17113#discussion-diff-48291261